integration/test_auth.py

```diff
@@ -177,6 +177,27 @@ def new_user_with_params():
     yield user
     auth.delete_user(user.uid)
 
+@pytest.fixture
+def imported_user_with_params():
+    random_id, email = _random_id()
+    phone = _random_phone()
+    user = auth.ImportUserRecord(
+        uid=random_id,
+        email=email,
+        phone_number=phone,
+        display_name='Random User',
+        photo_url='https://example.com/photo.png',
+        user_metadata=auth.UserMetadata(100, 150),
+        password_hash=b'password', password_salt=b'NaCl', custom_claims={'admin': True},
+        email_verified=True,
+        disabled=False,
+        provider_data=[auth.UserProvider(uid='test', provider_id='google.com')])
```

Comment:
Same comment as before: uid=test might cause this to fail if the test doesn't get properly cleaned up.